baseapp/a/a_test.go

```diff
@@ -0,0 +1,20 @@
+package a_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/cosmos/gogoproto/proto"
+	"github.com/stretchr/testify/require"
+
+	_ "github.com/cosmos/cosmos-sdk/x/bank/types"
+)
+
+// TestA is a temporary test to test file descriptors mismatch by importing
+// them 1 by 1.
+func TestA(t *testing.T) {
+	r, err := proto.MergedRegistry()
```

Comment:
I get this error:
```
        	Error:      	Received unexpected error:
        	            	got 2 file descriptor import path errors:
        	            	file name grpc/binlog/v1/binarylog.proto does not start with expected grpc/binarylog/v1/; please make sure your folder structure matches the proto files fully-qualified names
        	            	file name internal/errors.proto does not start with expected grpc/gateway/runtime/; please make sure your folder structure matches the proto files fully-qualified names
        	Test:       	TestA
```

I propose to remove the runtime import path checker in `gogoproto.MergedRegistry()`

Reply:
Logging a warning now.